library/spdm_common_lib/libspdm_com_crypto_service.c

```diff
@@ -607,12 +607,14 @@ bool libspdm_verify_peer_cert_chain_buffer_integrity(libspdm_context_t *spdm_con
         result = libspdm_verify_certificate_chain_buffer(
             spdm_context->connection_info.algorithm.base_hash_algo,
             spdm_context->connection_info.algorithm.base_asym_algo,
-            cert_chain_buffer, cert_chain_buffer_size, is_device_cert_model);
+            cert_chain_buffer, cert_chain_buffer_size,
+            !is_requester, is_device_cert_model);
```

Comment:
Yes you're right.
The parameter was intended to just tell if the function is verifying requester or responder cert.
I updated it to `is_requester_cert` to avoid the ambiguity.